### PR TITLE
Add `osu! Indonesia Tournament #13 (2023)` wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
Wiki article for the currently ongoing `osu! Indonesia Tournament #13 (2023)` tournament.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Nivalyx** [comment]
(i also included a fix for my [previous tourney pr](https://osu.ppy.sh/wiki/tournaments/oasis/1) where i forgot to delete a along with this)

File: wiki/Tournaments/OIT/2023/en.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player may be disqualified right away from the tournament and/or blacklisted from future iterations of the tournament by the tournament management.
+    - Usage of any tools or programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.
+
+### Tournament registration
+
+1. All interested players are required to register into the tournament individually through [this form](https://forms.gle/xzFzErPtCe989xHY6).
+   - In order to be eligible to play in the tournament, a player must:
+     - Have the ::{ flag=ID }:: Indonesian flag displayed on their profile, ***and***
+     - Be in possession of at least 6,000 pp in the osu! game mode at the time of registration.
+2. To ensure that all incoming registrations are serious and valid, every registered player will be checked in detail by the Tournament Management.
```
Comment (Walavouchey) [inline wiki/Tournaments/OIT/2023/en.md:115]:
```suggestion
2. To ensure that all incoming registrations are serious and valid, every registered player will be checked in detail by the tournament management.
```

File: wiki/Tournaments/OIT/2023/en.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+3. The list of players who are deemed to be eligible to compete in the tournament will be published by the Tournament Management after the Registration phase has ended.
```
Comment (Walavouchey) [inline wiki/Tournaments/OIT/2023/en.md:116]:
```suggestion
3. The list of players who are deemed to be eligible to compete in the tournament will be published by the tournament management after the registration phase has ended.
```

File: wiki/Tournaments/OIT/2023/en.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+3. Players **are not allowed** to ban any beatmaps in the Qualifiers.
+4. Players **are not allowed** to join (or register for) more than one Qualifier lobby.
+5. Based on their performance in the Qualifier, players will be ranked based on their Z-Sum value (which is the total amount of Z-Score across all eleven Qualifier beatmaps). A brief explanation on the concept of Z-Sum and Z-Score can be found [here](http://www.statisticshowto.com/probability-and-statistics/z-score/).
+6. 16 players with the **highest Z-Sum value** will advance to the knock-out stages. 
+   - If there are two (or more) players who share the same Z-Sum value, the player that holds the higher total score combined from all ten Qualifier beatmaps will be placed in the higher seed.
+7. Failure to attend in any of the predetermined Qualifier lobbies will result in an instant elimination from the tournament.
+
+#### Knock-out stage rules
+
+1. The 16 players who managed to get through from the Qualifiers will be matched to each other based on their Qualifiers seeding.
```
Comment (Walavouchey) [inline wiki/Tournaments/OIT/2023/en.md:134]:
```suggestion
1. The 16 players who managed to get through from the Qualifiers will be matched to each other based on their Qualifier seeding.
```

File: wiki/Tournaments/OIT/2023/en.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+2. Players will compete against each other using the double-elimination system.
+3. The double-elimination system works as follows:
+   - Players who lose in the upper bracket can still play again on the lower bracket.
```
Comment (Walavouchey) [inline wiki/Tournaments/OIT/2023/en.md:137]:
```suggestion
   - Players who lose in the upper bracket can still play again in the lower bracket.
```

**Comment by Nivalyx** [comment]
applied everything, thanks for the review as always @Walavouchey 👍

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**